homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -5,19 +5,39 @@
 import logging
 
 import voluptuous as vol
+from miio import (  # pylint: disable=import-error
+    AirFresh,
+    AirHumidifier,
+    AirPurifier,
+    Device,
+    DeviceException,
+)
+
+from miio.airfresh import (  # pylint: disable=import-error; pylint: disable=import-error
+    LedBrightness as airfreshLedBrightness,
+    OperationMode as airfreshOperationMode,
```

Comment:
I don't really like how this looks, so there'd be three options:
1. Import `miio.airfresh` and use `airfresh.LedBrightness` etc.?
2. Rename `airfreshLedBrightness` to `AirFreshLedBrightness` etc. to be consistent with the naming of the main class. Maybe in the future this should be changed to be such also in python-miio
3. Make these import first when they are used.

I personally think the second one is the nicest solution for the time being.

Reply:
@rytilahti I did option 2 as it was the easiest for me (still new to Python and HA)